lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -76,8 +88,11 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             // Telemetry manager only used to increment cacheHits here
             const serverTelemetryManager = this.initializeServerTelemetryManager(ApiId.acquireTokenSilent_silentFlow, silentRequest.correlationId);
             const silentAuthClient = await this.createSilentFlowClient(serverTelemetryManager, silentRequest.authority);
-            return await silentAuthClient.acquireCachedToken(silentRequest);
+            const cachedToken = await silentAuthClient.acquireCachedToken(silentRequest);
+            this.broadcastEvent(BroadcastEvent.ACQUIRE_TOKEN_SUCCESS, InteractionType.SILENT, cachedToken);
+            return cachedToken;
         } catch (e) {
+            this.broadcastEvent(BroadcastEvent.ACQUIRE_TOKEN_FAILURE, InteractionType.SILENT, null, e);
```

Comment:
This should be moved to the bottom of the catch block in `acquireTokenByRefreshToken` as this just represents a cache miss, not an actual failure

Reply:
Got it. Moved.